javacutil/src/org/checkerframework/javacutil/AnnotationBuilder.java

```diff
@@ -180,6 +180,30 @@ public void copyElementValuesFromAnnotation(
         return;
     }
 
+    /**
+     * Copies the specified element values from the given annotation, using the specified name
+     * mapping. If an element specified as a target in the mapping doesn't exist in the annotation
+     * to be built, an error is raised. If an element from the given annotation is not in the
+     * mapping, it is ignored.
+     *
+     * @param valueHolder the annotation that holds the values to be copied
+     * @param mapping mapping from source to target names
+     */
+    public void copyRenameElementValuesFromAnnotation(
+            AnnotationMirror valueHolder, Map<String, String> mapping) {
```

Comment:
`valueHolder` is the source, and `this` is the target.  I would indicate this more explicitly in the documentation or naming.